examples/mean/tests/Makefile

```diff
@@ -1,27 +1,23 @@
 
```

Comment:
Please don't change this file in this PR. Feel free to file another PR with these changes, but this PR should be limited to python / documentation changes.